src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private String[] color = {"green", "blue", "brown", "purple", "yellow"};
```

Comment:
The array name 'color' should be in plural form to represent that it contains multiple values, such as 'colors'.